modules/likes/jetpack-likes-master-iframe.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * This function needs to get loaded after the like scripts get added to the page.
+ */
+function jetpack_likes_master_iframe() {
+	$version = '20170607';
+	$in_jetpack = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? false : true;
+
+	$protocol = 'http';
+	if ( is_ssl() )
+		$protocol = 'https';
+
+	$_locale = get_locale();
+
+	// We have to account for w.org vs WP.com locale divergence
+	if ( $in_jetpack ) {
+		if ( ! defined( 'JETPACK__GLOTPRESS_LOCALES_PATH' ) || ! file_exists( JETPACK__GLOTPRESS_LOCALES_PATH ) ) {
+			return false;
+		}
+
+		require_once JETPACK__GLOTPRESS_LOCALES_PATH;
+
+		$gp_locale = GP_Locales::by_field( 'wp_locale', $_locale );
+		$_locale = isset( $gp_locale->slug ) ? $gp_locale->slug : '';
+	}
+
+	$likes_locale = ( '' == $_locale || 'en' == $_locale ) ? '' : '&amp;lang=' . strtolower( $_locale );
+
+	$src = sprintf(
+		'%1$s://widgets.wp.com/likes/master.html?ver=%2$s#ver=%2$s%3$s',
```

Comment:
This can be changed to always be `https`, as `widgets.wp.com` will redirect `http` requests.